sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -207,6 +208,7 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     // Returns true if the plan is supposed to be sorted.
     def isSorted(plan: LogicalPlan): Boolean = plan match {
       case _: Join | _: Aggregate | _: Generate | _: Sample | _: Distinct => false
+      case _: ShowColumnsCommand => true
```

Comment:
@cloud-fan @viirya Actually it does not break the test if we don't mark it as sorted. What happens is that, when we generate the expected output file, the results appear sorted like following:

```SQL
-- !query 7
SHOW COLUMNS IN showcolumn2 IN showdb
-- !query 7 schema
struct<col_name:string>
-- !query 7 output
month
price
qty
year
```
When i was going through the expected output file to make sure its correct, i noticed this as the above output would not be how it would be shown if i cut-paste the SQLs snippets from the test file and ran it in spark-sql shell. 

If you guys think its okay to have the output in sorted form in the expected file, then i will change it back.